### PR TITLE
Fix header for config-cheat-sheet

### PR DESCRIPTION
**Pull request by bkcsoft**
Hopefully this is correct :trollface:

### CODE REVIEW

File: content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 ---
 name: Config Cheat Sheet
```
Comment (tboerger) [inline content/doc/advanced/config-cheat-sheet.en-us.md]: Drop this line

File: content/doc/advanced/config-cheat-sheet.en-us.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+date: "2016-12-26T16:00:00+02:00"
+title: "Config Cheat Sheet"
+slug: "config-cheat-sheet"
+weight: 10
```
Comment (tboerger) [inline content/doc/advanced/config-cheat-sheet.en-us.md]: Same order as hacking on gitea?

**Comment by tboerger** [comment]
LGTM

**Comment by bkcsoft** [comment]
whomever LG_TM, _SQUASH_ 😂

**Comment by andreynering** [comment]
LGTM

**Pull request merged**